pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/configure-bastion/index.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: layout.pug
+navigationTitle: Configure bastion hosts
+title: Configure bastion hosts
+menuWeight: 60
+excerpt: Configure bastion hosts for access to the DKP cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+A **bastion host** is a server that provides a defensive barrier to protect access to a private network from an external network, such as the internet.
+
+Bastion hosts are important because, for security reasons, you often want to deploy all nodes in a cluster on a private network.
+Deploying the cluster nodes on a private network protects cluster resources from outside access because the nodes cannot be reached directly from any external network.
+
+To enable installation of cluster resources from an external network, you can configure a [bastion host][bastion_host].
+
+With a bastion host, you can use a secure shell (SSH) session to connect to cluster nodes from outside of the private network.
+
+If you want to configure a bastion host for a DKP cluster deployed on an Amazon Web Services cloud instance, you can specify the bastion host information using the `ClusterProvisioner` section in the `cluster.yaml` file.
+
+For example, you can configure bastion host settings using a special [node pool][node_pool] section similar to the following:
```

Comment:
We should probably change this link to something more recent ... such as: 
/dkp/konvoy/2.2/choose-infrastructure/aws/nodepools/

Reply:
Thanks for updating, resolved.